app/main/helpers/frameworks.py

```diff
@@ -39,19 +39,17 @@ def get_required_fields(all_fields, answers):
     ])
     required_fields -= optional_fields
     #  If you answered other to question 19 (trading status)
-    if answers.get('SQ1-1ci') == 'other (please specify)':
+    if 'SQ1-1ci' in page_fields and answers.get('SQ1-1ci') == 'other (please specify)':
         required_fields.add('SQ1-1cii')
-
     #  If you answered yes to question 27 (non-UK business registered in EU)
-    if answers.get('SQ1-1i-i', False):
+    if 'SQ1-1i-i' in page_fields and answers.get('SQ1-1i-i', False):
         required_fields.add('SQ1-1i-ii')
-
     #  If you answered 'licensed' or 'a member of a relevant organisation' in question 29
-    if len(answers.get('SQ1-1j-i', [])) > 0:
+    if 'SQ1-1j-i' in page_fields and len(answers.get('SQ1-1j-i', [])) > 0:
```

Comment:
Without this check an error is generated for that field on every page of the split-up declaration, even though the question wasn't on the page.